tests/network/vmi_networking.go

```diff
@@ -761,23 +761,6 @@ var _ = Describe(SIG("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:
 			)
 		})
 	})
-
-	Context("VirtualMachineInstance with TX offload enabled", func() {
-		It("[test_id:1781]should have tx checksumming enabled on interface serving dhcp", func() {
-			vmi, err := virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(
-				context.Background(),
-				libvmifact.NewAlpine(libvmi.WithMemoryRequest("1024M")),
-				metav1.CreateOptions{},
-			)
-			Expect(err).ToNot(HaveOccurred())
-			libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
-			output := libpod.RunCommandOnVmiPod(
-				vmi,
-				[]string{"/bin/bash", "-c", "/usr/sbin/ethtool -k k6t-eth0|grep tx-checksumming|awk '{ printf $2 }'"},
```

Comment:
Checking the history [1], ethtool was introduced to virt-launcher just for this test. Since we are removing the test, could you please remove the dependency as well?
 
[1] https://github.com/kubevirt/kubevirt/pull/6579